web/src/app/modules/shared/components/presentation/select-file/select-file.component.ts

```diff
@@ -13,8 +13,16 @@ import {
 } from '@angular/core';
 import { AbstractViewComponent } from '../../abstract-view/abstract-view.component';
 import { SelectFileView } from '../../../models/content';
-import { WebsocketService } from '../../../../../data/services/websocket/websocket.service';
 import { ActionService } from '../../../services/action/action.service';
+import { ElectronService } from '../../../services/electron/electron.service';
+
+type File = {
+  name: string;
+  type: string;
+  path: string;
```

Comment:
`path` is not passed from browser for security reason. This limits action consumer ability to read the file (see below)